homeassistant-supervised/DEBIAN/postinst

```diff
@@ -163,21 +163,21 @@ systemctl start hassio-supervisor.service
 info "Installing the 'ha' cli"
 chmod a+x "${PREFIX}/bin/ha"
 
-# Switch to cgroup v1
+# Switch to cgroup v2
 if [ -f /etc/default/grub ]
 then
-    if ! grep -q "systemd.unified_cgroup_hierarchy=false" /etc/default/grub; then
-        info "Switching to cgroup v1"
+    if grep -q "systemd.unified_cgroup_hierarchy=false" /etc/default/grub; then
+        info "Switching to cgroup v2"
         cp /etc/default/grub /etc/default/grub.bak
-        sed -i 's/^GRUB_CMDLINE_LINUX_DEFAULT="/&systemd.unified_cgroup_hierarchy=false /' /etc/default/grub
+        sed -i 's/ systemd\.unified_cgroup_hierarchy=false//' /etc/default/grub
```

Comment:
We've added the space at the end above it seems, so maybe we should also remove the one at the end here? :thinking: 

We could just not remove the space, or maybe optionally?

Reply:
I figured I'd remove the space since we added it to keep everything clean thus why I added the space at the beginning of the sed search and replace, it seems to match and replace removing the space in my testing, is there something I should be doing differently?

Reply:
The old `sed` added the space after the entry. So my `/etc/default/grub` looks like this:

```
...
GRUB_CMDLINE_LINUX_DEFAULT="systemd.unified_cgroup_hierarchy=false quiet"
...
```

So if you want to get rid of the space created by the original `sed` command, you have to drop the one at the end:

```suggestion
        sed -i 's/systemd\.unified_cgroup_hierarchy=false //' /etc/default/grub
```